dali/test/python/test_external_source_dali.py

```diff
@@ -522,7 +501,7 @@ def pipeline():
         f"but the {pruned_str} not used."
     )
     with assert_warns(Warning, glob=expected_error_msg):
-        pipe.build()
+        pipe.run()
```

Comment:
We want to know that the error occurs in `build` phase.
```suggestion
        pipe.build()
```